### ISSUE TITLE
Appimage 1.30: execv error: No such file or directory

### ISSUE THREAD

**Issue by UltraBlackLinux** [HACKERALERT/Picocrypt#130]
Hey there,
It seems that with the newest update this tool does not execute on linux, or rather the appimage doesn't.
I just confirmed that it works on 1.29; 1.30 just exits with `execv error: No such file or directory`. I don't know what the launch script is, but it's probably supposed to say `execvp` (Why is there no branch for appimages or something like that? The process is rather intransparent currently.
Please fix this!
Thanks!

**Comment by HACKERALERT** [comment]
I switched from the official AppImage to go-appimage between 1.29 and 1.30, so that may be the reason. This was because Ubuntu 22.04 doesn't have libfuse2 and the official AppImage builder creates AppImages that don't run. But if using the alternative go-appimage causes issues, I might as well go back to the more stable and official AppImageKit for 1.31.

Linux isn't the primary platform for Picocrypt, so I don't feel a need to make a dedicated branch. Still though, I have the files and scripts for building the AppImage openly available: https://github.com/HACKERALERT/Picocrypt/issues/84#issuecomment-1256840095

TL;DR, this issue will be fixed in the 1.31 release as I will go back to AppImageKit.

**Comment by HACKERALERT** [comment]
@UltraBlackLinux I've just release v1.31, which uses the same AppImage builder as 1.29, and should work properly. I'll close this issue since the problem should be solved, but please feel free to reopen it. Cheers!
